<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableList">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>10</y>
      <width>441</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>711</width>
      <height>561</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="verticalSpacing">
      <number>20</number>
     </property>
     <item row="1" column="2">
      <widget class="QPushButton" name="addCustomer">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Add Customer</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>528</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QListWidget" name="waitingList"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>42</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHistory">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Us"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit </string>
    </property>
    <addaction name="actionAdd_Customer"/>
    <addaction name="actionTable_History"/>
   </widget>
   <addaction name="menuHistory"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout_Us">
   <property name="icon">
    <iconset resource="tableListResources.qrc">
     <normaloff>:/icons/aboutIcon.png</normaloff>:/icons/aboutIcon.png</iconset>
   </property>
   <property name="text">
    <string>About Us</string>
   </property>
  </action>
  <action name="actionAdd_Customer">
   <property name="icon">
    <iconset resource="tableListResources.qrc">
     <normaloff>:/icons/customerIcon.png</normaloff>:/icons/customerIcon.png</iconset>
   </property>
   <property name="text">
    <string>Add Customer</string>
   </property>
  </action>
  <action name="actionTable_History">
   <property name="icon">
    <iconset resource="tableListResources.qrc">
     <normaloff>:/icons/historyIcon.png</normaloff>:/icons/historyIcon.png</iconset>
   </property>
   <property name="text">
    <string>Table History</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="tableListResources.qrc">
     <normaloff>:/icons/exitIcon.png</normaloff>:/icons/exitIcon.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="tableListResources.qrc"/>
 </resources>
 <connections/>
</ui>
